tests/test_ptc.py

```diff
@@ -281,6 +281,44 @@ def test_meanVarMeasurement(self):
         self.assertLess(self.flatWidth - np.sqrt(varDiff), 1)
         self.assertLess(self.flatMean - mu, 1)
 
+    def test_meanVarMeasurementSomeNan(self):
+        task = self.defaultTask
+        self.flatExp1.image.array[20:30] = np.nan
+        self.flatExp2.image.array[20:30] = np.nan
+
+        mu, varDiff, _ = task.measureMeanVarCov(self.flatExp1, self.flatExp2)
+
+        expectedMu1 = np.nanmean(self.flatExp1.image.array)
+        expectedMu2 = np.nanmean(self.flatExp2.image.array)
+        expectedMu = 0.5*(expectedMu1 + expectedMu2)
+
+        # Now the variance of the difference. First, create the diff image.
+        im1 = self.flatExp1.maskedImage
+        im2 = self.flatExp2.maskedImage
+
+        temp = im2.clone()
+        temp *= expectedMu1
+        diffIm = im1.clone()
+        diffIm *= expectedMu2
+        diffIm -= temp
+        diffIm /= expectedMu
+
+        expectedVar = 0.5*np.nanvar(diffIm.image.array)
+
+        self.assertLess(np.sqrt(expectedVar) - np.sqrt(varDiff), 1)
+        self.assertLess(expectedMu - mu, 1)
+
+    def test_meanVarMeasurementAllNan(self):
+        task = self.defaultTask
+        self.flatExp1.image.array[:, :] = np.nan
+        self.flatExp2.image.array[:, :] = np.nan
+
+        mu, varDiff, covDiff = task.measureMeanVarCov(self.flatExp1, self.flatExp2)
+
+        self.assertTrue(mu is np.nan)
+        self.assertTrue(varDiff is np.nan)
+        self.assertTrue(covDiff is np.nan)
```

Comment:
I think these should all be changed to `self.assertTrue(np.isnan(mu))` etc because if those functions above returned `float('nan')` instead then this would fail, and that would be annoying and hard to debug.